### DISCUSSION TITLE
Solution for regular status calls

### DISCUSSION THREAD

**Discussion by pks-t** [libgit2/libgit2#5884] [General]
Our intention is to hold regular status calls in which at least the core members and, if desired, additional participants can discuss the current state of affairs like issues, pull requests etc. To do this, we need to find a solution which provides group conferencing, accessible by everybody and ideally even open.

- The obvious choice for group conferencing would be Skype, but due to that not being open I'm not too keen on using that.
- Slack provides video conferencing when you have a paid plan. While that is at least not as bad as the requirement to have a Microsoft account, Slack itself is still not free. We do not have a paid plan, though, making this a non-option for now.
- Matrix/Riot.im is an open and decentralized protocol. While its main focus is to provide messaging functionality, it claims to support audio/video conferencing as well. This is obviously the best choice with regards to openness, but I don't yet have any experience with its conferencing feature.

**Comment by carlosmn** [comment]
We don't need decentralized as a feature, right? If https://meet.jit.si works for you that would be a pretty good option. It's open but they provide a hosted solution and it runs in the browser (but they do have apps to download).

For just audio, mumble would probably work.

**Comment by ethomson** [comment]
I'd like to move this forward.  I've already been having 1:1 discussions with people about libgit2 or LibGit2Sharp and would like to move this to a public forum where everybody can benefit.

Can we get some options to move this forward?  Surely other open source projects are doing this and have similar requirements.  What are they using?

**Comment by pks-t** [comment]
I'd be fine with Jitsi, I guess. Still need to figure out how to get it working here, but let's just use that at least for the first meeting and settle on a date. We can still make the conferencing software one of the topics of that call.

One thing that just occurred to me: is Jitsi able to record the conference? That's definitely a thing we should do to enable other people to listen to that conference in case they were not able to attend.

**Comment by carlosmn** [comment]
It supports streaming to YouTube's livestream service, which does record the streams. There's a [few steps](https://jitsi.org/live-streaming-and-recording-a-jitsi-conference/) to set it up the first time but after that it should be fairly easy to do. This should be a fairly reliable way of storing the video, even if the sound might not end up as well as if everyone recorded everything and we mixed it together but we don't have time to do that either way.

Jitsi is also a client-side program if you prefer running that to the browser as an app platform.

**Comment by pks-t** [comment]
Let's finally drive forward the first call and settle on a date. Would one of June 3rd/4th or June 9th/10th work for you guys?

**Comment by carlosmn** [comment]
I'm away 3rd/4th, but can do 9th/10th.

**Comment by pks-t** [comment]
@ethomson, would that weekend on June 9th/10th suit you?

**Comment by ethomson** [comment]
Yes, I should be able to find some time.

**Comment by pks-t** [comment]
Just to keep this rolling: I'd propose either Saturday somewhere
between 9am and 1pm CEST or Sunday between 2pm and 10pm CEST.
I'm not available outside of those two timeframes.

**Comment by carlosmn** [comment]
I am available for both those time ranges.

**Comment by ethomson** [comment]
Sorry, I forgot about this until just now.  Tomorrow, then?  Maybe 15:00 CEST?

**Comment by pks-t** [comment]
No worries. 15:00 CEST works for me.

**Comment by carlosmn** [comment]
Since we never actually agreed on the time and it didn't show up on my
calendar I ended up on an impromptu trip so I won't be able to make it. But
we can try Monday or Tuesday?

**Comment by pks-t** [comment]
Fair enough. I'm flexible tomorrow, I guess every timeslot
between 10:30am and 10pm would work for me.

Our process how we organize that call definitely needs to improve
for the next one ;) Especially if we want to allow external
people to participate.

**Comment by carlosmn** [comment]
Let's do it earlier rather than later as I have to catch a plane in the
evening. Starting before 2pm should give me enough buffer even if we go
long. I expect to be available from 10:30 onwards too.
